WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -1025,7 +1058,7 @@ function OptionsPrivate.SortDisplayButtons(filter, overrideReset, id)
       button:ResetLinkedAuras()
     end
     for id, aura in pairs(WeakAurasSaved.displays) do
-      if not aura.ignoreWagoUpdate and aura.url and aura.url ~= "" then
+      if (aura.ignoreWagoUpdate and aura.ignoreWagoUpdate == true or not aura.ignoreWagoUpdate) and aura.url and aura.url ~= "" then
```

Comment:
I don't understand this change.

Reply:
i might misunderstand lua syntax but "not aura.ignoreWagoUpdate" should return both for  aura.ignoreWagoUpdate being true or false right? so simply checking for that won't work if we just set the key to false

Reply:
not aura.ignoreWagoUpdate will return true for "nil" and "false", and the code inside the if should only be run if ignoreWagoUpdate is nil or false, so that looks correct without your change.

I'm not sure I understand your confusion though.

Reply:
true ur totally correct and i'm not sure what i thought when doing that except it being late and me being tired